src/Etterna/Actor/Gameplay/Foreground.cpp

```diff
@@ -6,6 +6,8 @@
 #include "RageUtil/Graphics/RageTextureManager.h"
 #include "RageUtil/Utils/RageUtil.h"
 #include "Etterna/Models/Songs/Song.h"
+#include "Etterna/Singletons/GameState.h"
```

Comment:
What's this `#include` for?

Reply:
This no longer applies. It's apart of the bad merge which has been force overwritten.